js/main.js

```diff
@@ -144,7 +132,7 @@ function entryFormViewSwap() {
   viewSwap('entry-form');
 }
 
-$entriesAnchor.addEventListener('click', entryViewSwap);
+$entriesAnchor.addEventListener('click', viewSwap('entries'));
```

Comment:
You can't call `viewSwap` like this, because it's going to immedieatly execute once it hits this block of code. You need to create a callback function, and call `viewSwap` inside of there.

```
$entriesAnchor.addEventListener('click', function() {
  viewSwap('entries')
}
```